site.conf

```diff
@@ -65,14 +83,19 @@
           '41a126dfdd336be67005d15c8b4776775e2bbf1ec16fbe594e7a7ee377c0886d', -- Tux
           '4611823f7f050d465252a91c24c4db8887ebd98ef880df92329212919c8497bd', -- Johann Wagner
           '4d9d050abf0b5300e8c893aa2da7cf8aac32a86a7ee7e8bbde286de74605ad2a', -- Franz Kuntke
+          '9a5ceee2ba63dbe8f8d1db3dc3e90050a08b54c9c37858722be9f082841d857f', -- Juergen S.
         },
 			},
 		},
 	},
 
   opkg = {
 		extra = {
-			modules = 'http://experimental-babel.updates.firmware.ffmd/modules/gluon-%GS-%GR/%S',
+			modules = {
+				'http://stable-babel.updates.firmware.ffmd/modules/gluon-%GS-%GR/%S',
+				'http://test-babel.updates.firmware.ffmd/modules/gluon-%GS-%GR/%S',
+				'http://experimental-babel.updates.firmware.ffmd/modules/gluon-%GS-%GR/%S',
```

Comment:
we'll need a separate issue for creating the dns record for test-babel.updates.firmware.ffmd @penguineer What do you think? should we lifecycle pdns and create the record then or the other way around?